ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/get_anta_results.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+import logging
+from asyncio import run
+from json import loads
+
+from ansible.plugins.connection import ConnectionBase
```

Comment:
Can we avoid importing ansible specific things here?

Reply:
not really because we want an AnsibleEOSDevice class and we need to pass it the Connection from the action plugin to instantiate it - though we can import it only on TYPE_CHECKING for this specific file but we cannot avoid this for the Ansible part.

That being said when moving to pyavd, we will need to modify this function to use AsyncEOSDevice from ANTA as we won't have the Ansible connection to leverage for standalone pyavd so we will need to find a way to give another type of input and not the Connection for this. (or make it optional).

Probably worth discussing on a call